pkg/webui/console/views/gateway-collaborator-edit/index.js

```diff
@@ -0,0 +1,268 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { connect } from 'react-redux'
+import bind from 'autobind-decorator'
+import { Container, Col, Row } from 'react-grid-system'
+import * as Yup from 'yup'
+import { replace } from 'connected-react-router'
+
+import { withBreadcrumb } from '../../../components/breadcrumbs/context'
+import Breadcrumb from '../../../components/breadcrumbs/breadcrumb'
+import sharedMessages from '../../../lib/shared-messages'
+import Form from '../../../components/form'
+import Field from '../../../components/field'
+import Button from '../../../components/button'
+import Spinner from '../../../components/spinner'
+import ModalButton from '../../../components/button/modal-button'
+import Message from '../../../lib/components/message'
+import FieldGroup from '../../../components/field/group'
+import IntlHelmet from '../../../lib/components/intl-helmet'
+import toast from '../../../components/toast'
+import SubmitBar from '../../../components/submit-bar'
+
+import { getGatewayCollaboratorsList } from '../../store/actions/gateway'
+import { getGatewaysRightsList } from '../../store/actions/gateways'
+import api from '../../api'
+
+import style from './gateway-collaborator-edit.styl'
+
+// TODO: Move this to checkbox group later, see https://github.com/TheThingsNetwork/lorawan-stack/issues/189
+const UNIVERSAL_APPLICATION_RIGHTS = [ 'RIGHT_APPLICATION_ALL', 'RIGHT_ALL' ]
```

Comment:
just a remainder
after https://github.com/TheThingsNetwork/lorawan-stack/pull/817 it wont be needed